scripts/cmake/add_copy_assets.cmake

```diff
@@ -0,0 +1,69 @@
+
+function(add_copy_assets)
+
```

Comment:
As per the project structure recommended in the Moder CMake guide, it would be better to put these files into `cmake` at the root level.

https://cliutils.gitlab.io/modern-cmake/chapters/basics/structure.html

`scripts` should only contain actual scripts and tools we can execute from the CLI.